tests/integration/components/bs-modal-test.js

```diff
@@ -151,4 +151,28 @@ module('Integration | Component | bs-modal', function(hooks) {
     assert.dom('.modal').hasAttribute('role', 'alert');
     assert.dom('.modal').hasAttribute('data-test');
   });
+
+  test('it keeps itself visible when mouse click is started from inside of the modal \
+    and dragged outside of the modal', async function(assert) {
+    await render(hbs`
+      <BsModal>
+        template block text
+      </BsModal>
+    `);
+    await triggerEvent('.modal-content', 'mousedown');
+    await triggerEvent('.modal', 'mouseup');
```

Comment:
I think we should add `await click('.modal');` here, as that is the sequence of events, see https://javascript.info/mouse-events-basics#complex-events. Also only the click event would have triggered the wrong behaviour previously. Could you please try to confirm that the test fails without the changes in the component?

Reply:
`click` test helper provided by `@ember/test-helpers` would trigger another `mousedown` event. So I guess we would need to do:

```js
    await triggerEvent('.modal-content', 'mousedown');
    await triggerEvent('.modal', 'mouseup');
    await triggerEvent('.modal', 'click');
```

Reply:
@simonihmig Yeah, when I revert my changes locally on the component, the test case with name `it keeps itself visible when mouse click is started from inside of the modal and dragged outside of the modal` is failing locally.

@jelhan Thank you for your help, I updated the test case accordingly.